packages/store-api/src/typeDefs/pageInfo.graphql

```diff
@@ -0,0 +1,8 @@
+type StorePageInfo {
+  hasNextPage: Boolean!
+  hasPreviousPage: Boolean!
+  startCursor: String!
+  endCursor: String!
+  # Total number of items, not pages
```

Comment:
What do you mean by items? Products? If so, why not make it more explicit?

Reply:
It's because we are in the context of pagination. When handling with pagination we have mainly two entities, a page and an item. I just wanted to explicit the fact that these are total number of items, not pages. This type is used for all pagination, a.k.a, products and collections. 

What do you think about something like?
```suggestion
  # Total number of items (products/collections), not pages
```

Reply:
I see! I understood the context because of the other fields in the type hahahah.
But this suggestion already solves it, that's great!